src/main/java/persistence/sql/model/Column.java

```diff
@@ -87,12 +99,10 @@ public String getName() {
         return name;
     }
 
-    @Override
-    public SqlType getType() {
-        return type;
+    public Optional<SqlType> getType() {
+        return Optional.ofNullable(type);
```

Comment:
```suggestion
    public SqlType getType() {
        if (type == null) {
            throw new IllegalStateException("Column type is not set");
        }
        return type;
```
Optional 사용도 좋지만 공통적인 예외를 응답하게 하려면 이런 처리가 더 적합해 보이는데 어떻게 생각하세요?